src/ui/trigger_editor.ui

```diff
@@ -704,7 +704,7 @@
                  <item>
                   <widget class="QToolButton" name="button_toggleSearchAreaResults">
                    <property name="toolTip">
-                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggles the display of the search results area.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                    <string>Toggles the display of the search results area.</string>
```

Comment:
Not judging by the above result where it has rather poorly split the sentence into 2-3 words per line and added a blank one at the end, no...

I'm just correcting this one particular instance because it looks poor.